arxiv/base/templates/base/base.html

```diff
@@ -8,7 +8,7 @@
   </head>
   <body>
   {% import "base/macros.html" as macros %}
-  <noscript><img src="//webanalytics.library.cornell.edu/piwik.php?idsite=538&rec=1" style="border:0;" alt="" /></noscript>
+  <noscript><p><img src="https://webstats.arxiv.org/matomo.php?idsite=1&amp;rec=1" style="border:0;" alt="" /></p></noscript>
```

Comment:
I'm mildly curious (but not concerned) about the `p` tag here :-)

Reply:
Yeah, I looked at that too - this was stock code from Matomo.
My best guess is that it is for strict HTML validators, after a little bit of research.
Won't hurt, it doesn't generate unless JavaScript is disabled - might be worth checking to see if an auto-margin gets applied on the p tag with a zero-size image but I suspect that it won't.